x-pack/plugins/reporting/server/plugin.ts

```diff
@@ -38,7 +38,8 @@ export class ReportingPlugin
     const basePath = http.basePath.get;
 
     // async background setup
-    buildConfig(initContext, core, this.logger).then((config) => {
+    (async () => {
```

Comment:
This felt better/easier to write more async setup tasks with in the future since we have our own .catch in case issues should arise

Reply:
I like it!